packages/fast-components-react-base/src/slider/slider.props.ts

```diff
@@ -0,0 +1,112 @@
+import * as React from "react";
+import {
+    ManagedClasses,
+    SliderClassNameContract,
+} from "@microsoft/fast-components-class-name-contracts-base";
+import { SliderState, SliderThumb } from "./slider";
+
+export interface SliderManagedClasses extends ManagedClasses<SliderClassNameContract> {}
+export interface SliderUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
+
+export enum SliderOrientation {
+    horizontal = "horizontal",
+    vertical = "vertical",
+}
+
+export enum SliderMode {
+    singleValue = "singleValue",
+    adustUpperValue = "adustUpperValue",
+    adustLowerValue = "adjustLowerValue",
+    adjustBoth = "adjustBoth",
+}
+
+export interface SliderRange {
+    minValue: number;
+    maxValue: number;
+}
+
+export interface SliderHandledProps extends SliderManagedClasses {
+    /**
+     * The slider orientation, default is horizontal
+     */
+    orientation?: SliderOrientation;
+
+    /**
+     * The slider mode
+     */
+    mode?: SliderMode;
+
+    /**
+     * Custom function to render the thumb of the control
+     */
+    thumb?: (
```

Comment:
could this be control? i don't know what thumb means

Reply:
"control" is way to overloaded.  "thumb" is the generic term for the thing you drag around on a slider.  Not sure what would be clearer?  Peanut gallery feedback?

Reply:
"Thumb" seems like a standard term to me - https://www.w3.org/TR/wai-aria-practices/examples/slider/multithumb-slider.html

Reply:
I'd say "thumb" is relatively standard - I'd support it leaving as-is